### DISCUSSION TITLE
Flutter Network Inspector and/or Firebase Performance Monitoring

### DISCUSSION THREAD

**Discussion by simplenotezy** [cfug/dio#2107] [General]
I noticed that both Flutter Network Inspector and the Firebase Performance Monitoring does not work when using dio - is there a way to enable this by this library?

**Comment by kuhnroyal** [comment]
This may already work for some `http` package based adapters but everything else would need to be implemented using the https://pub.dev/packages/vm_service package.

**Reply by kuhnroyal** [reply]
So from what I can tell, the default `IoHttpClientAdapter` already logs events to the network inspector.
Which platforms/adapters are not working for you?

**Reply by kuhnroyal** [reply]
For Firebase you should consider creating your own interceptor.
